tests/typ/math/matrix.typ

```diff
@@ -1,86 +1,5 @@
 // Test matrices.
 
----
```

Comment:
why delete this test?

Reply:
Don't worry, there is a bug I have been trying to track down, they'll be back, it's temporary while I maked it as draft.